Apps/Examples/Examples/All Examples/CustomAnchoredSymbolExample.swift

```diff
@@ -0,0 +1,321 @@
+        }
+
+        var shapeLayer: SymbolLayer
+
+        if mapView.mapboxMap.style.layerExists(withId: CustomAnchoredSymbolExample.symbols) {
+            shapeLayer = try! mapView.mapboxMap.style.layer(withId: CustomAnchoredSymbolExample.symbols) as SymbolLayer
+        } else {
+            shapeLayer = SymbolLayer(id: CustomAnchoredSymbolExample.symbols)
+        }
+
+        shapeLayer.source = CustomAnchoredSymbolExample.symbols
+
+        shapeLayer.textField = .expression(Exp(.get) {
+            "text"
+        })
+
+        shapeLayer.iconImage = .expression(Exp(.get) {
+            "imageName"
+        })
+
+        shapeLayer.textColor = .expression(Exp(.switchCase) {
+            Exp(.any) {
+                Exp(.get) {
+                    "highlighted"
+                }
+            }
+            UIColor.white
+            UIColor.black
+        })
+
+        shapeLayer.textSize = .constant(16)
+        shapeLayer.iconTextFit = .constant(.both)
+        shapeLayer.iconAllowOverlap = .constant(true)
+        shapeLayer.textAllowOverlap = .constant(true)
+        shapeLayer.textJustify = .constant(.left)
+        shapeLayer.symbolZOrder = .constant(.auto)
+        shapeLayer.symbolSortKey = .expression(Exp(.get) { "sortOrder" })
+        shapeLayer.textFont = .constant(["DIN Pro Medium"])
+
+        let expression = Exp(.switchCase) {
+            Exp(.eq) {
+                Exp(.get) { "tailPosition" }
+                0
+            }
+            "bottom-left"
+            Exp(.eq) {
+                Exp(.get) { "tailPosition" }
+                1
+            }
+            "bottom-right"
+            Exp(.eq) {
+                Exp(.get) { "tailPosition" }
+                2
+            }
+            "bottom"
+            ""
+        }
+        shapeLayer.textAnchor = .expression(expression)
+
+        let offsetExpression = Exp(.switchCase) {
+            Exp(.eq) {
+                Exp(.get) { "tailPosition" }
+                0
+            }
+            Exp(.literal) {
+                [0.7, -2.0]
+            }
+            Exp(.eq) {
+                Exp(.get) { "tailPosition" }
+                1
+            }
+            Exp(.literal) {
+                [-0.7, -2.0]
+            }
+            Exp(.eq) {
+                Exp(.get) { "tailPosition" }
+                2
+            }
+            Exp(.literal) {
+                [-0.2, -2.0]
+            }
+            Exp(.literal) {
+                [0.0, -2.0]
+            }
+        }
+        shapeLayer.textOffset = .expression(offsetExpression)
+
+        try? mapView.mapboxMap.style.addLayer(shapeLayer)
+    }
+}
+
+extension UIImage {
+    // Produce a copy of the image with tint color applied.
+    // Useful for deployment to iOS versions prior to 13 where tinting support was added to UIImage natively.
+    func tint(_ tintColor: UIColor) -> UIImage {
```

Comment:
```suggestion
    func tinted(with tintColor: UIColor) -> UIImage {
```

* `tinted` vs `tint`: The latter implies mutation. The former implies the creation of a new object.
* `_` vs `with`: without the argument label, at the point of use, it reads as if you're tinting the color rather than the image.